shavar/e2e-test/prefs.ini

```diff
@@ -0,0 +1,65 @@
+################################
+# global
+################################
+
+[DEFAULT]
+browser.startup.homepage = http://itisatrap.org/firefox/its-a-tracker.html 
+browser.safebrowsing.debug = true
+browser.safebrowsing.enabled = false
+browser.safebrowsing.malware.enabled = false
+privacy.trackingprotection.enabled = true
+browser.safebrowsing.provider.mozilla.nextupdatetime = 1
+
+#private.browsing.autostart = true
+
+
+################################
+# test-environments
+################################
+
+[stage]
+browser.safebrowsing.provider.mozilla.gethashURL = https://shavar.stage.mozaws.net/gethash?client=SAFEBROWSING_ID&appver=%VERSION%&pver=2.2
+browser.safebrowsing.provider.mozilla.updateURL = https://shavar.stage.mozaws.net/downloads?client=SAFEBROWSING_ID&appver=%VERSION%&pver=2.2
+
+[pre-prod]
+browser.safebrowsing.provider.mozilla.gethashURL = https://shavar.prod.mozaws.net/gethash?client=SAFEBROWSING_ID&appver=%VERSION%&pver=2.2
+browser.safebrowsing.provider.mozilla.updateURL = https://shavar.prod.mozaws.net/downloads?client=SAFEBROWSING_ID&appver=%VERSION%&pver=2.2
+
+[prod]
+browser.safebrowsing.provider.mozilla.gethashURL = https://shavar.services.mozilla.com/gethash?client=SAFEBROWSING_ID&appver=%VERSION%&pver=2.2
+browser.safebrowsing.provider.mozilla.updateURL = https://shavar.services.mozilla.com/downloads?client=SAFEBROWSING_ID&appver=%VERSION%&pver=2.2
+
+
+################################
+# pref-sets by file groupings
+################################
+
+[mozpub]      # PROD ONLY
+urlclassifier.trackingTable = test-track-simple,mozpub-track-digest256
+urlclassifier.trackingWhitelistTable = test-trackwhite-simple,mozpub-trackwhite-digest256
```

Comment:
This should be just `"test-trackwhite-simple"` because `mozpub-trackwhite-digest256` doesn't exist.